staging/src/k8s.io/client-go/pkg/apis/clientauthentication/types.go

```diff
@@ -58,20 +58,20 @@ type ExecCredentialStatus struct {
 	ExpirationTimestamp *metav1.Time
 	// Token is a bearer token used by the client for request authentication.
 	// +optional
-	Token string
+	Token string `datapolicy:"token"`
 	// PEM-encoded client TLS certificate.
 	// +optional
 	ClientCertificateData string
 	// PEM-encoded client TLS private key.
 	// +optional
-	ClientKeyData string
+	ClientKeyData string `datapolicy:"secret-key"`
 }
 
 // Response defines metadata about a failed request, including HTTP status code and
 // response headers.
 type Response struct {
 	// Headers holds HTTP headers returned by the server.
-	Header map[string][]string
+	Header map[string][]string `datapolicy:"token"`
```

Comment:
by the server, not the client.

Reply:
Is there possibility of those headers having any sensitive information?

Reply:
am not aware of any such case.

Reply:
This is still marked as token